src/integTest/resources/sample.servlet/verifyTimeoutSuccessAppsTest.gradle

```diff
@@ -34,7 +34,7 @@ ext {
     testServerHttpsPort = 9443
 
     // This is set in the ibm-web-ext.xml file
-    warContext = 'myLibertyApp'
+    warContext = 'myLibertyApp1'
```

Comment:
I remember that myLibertyApp1 was set in server.xml, but I don't see why this had to change now?  I.e., what is really different?